### DISCUSSION TITLE
Support for new Layout resizing options in the Sections type of dashboard

### DISCUSSION THREAD

**Discussion by grizzlyzp1** [Clooos/Bubble-Card#632] [Feature requests]
Super curious if the author is planning to support resizing Bubble Cards via Layouts options released with HA 2024.7
If not, is there a way to do it via yaml for instance to make it 2 rows high so I can fit more sub buttons on it.
Love this project.

**Comment by Clooos** [comment]
Hi, I was thinking of adding new card layouts directly in Bubble Card but this is a good idea indeed!
